.github/workflows/ci.yaml

```diff
@@ -34,7 +34,28 @@ jobs:
         run: npm ci
       - name: Build
         run: npm run build
+      - name: Archive artifact
+        uses: actions/upload-artifact@v1
+        with:
+          name: build
+          path: build
       - name: Flow
         run: npx flow
       - name: Test
         run: npm test
+  deploy:
+    name: Deploy
+    needs: build
+    runs-on: ubuntu-18.04
+    steps:
```

Comment:
Please add a check here, above `steps`, to only run on the project repo, the `dev-0.1` branch, and only on `push` events:

```
if: (github.repository == 'codelearncreate/c2lc-coding-environment') && (github.ref == 'refs/heads/dev-0.1') && (github.event_name == 'push')
```